src/game_stats.py

```diff
@@ -155,7 +155,10 @@ def _create_msg(self):
         return msg
 
     def _send_status_update(self):
-        msg = f"{self._config.general['name']}: Status Report\n{self._create_msg()}\nVersion: {__version__}"
+        if self._config.general["message_api_type"] == "generic_api":
+            msg = f"Status Report\n{self._create_msg()}\nVersion: {__version__}"
+        else:
+            msg = f"{self._config.general['name']}: Status Report\n{self._create_msg()}\nVersion: {__version__}"
```

Comment:
I would rather remove the name here and add it to the DiscordEmbeds class instead of this if clause

Reply:
no prob, will do.